CMakeLists.txt

```diff
@@ -1,24 +1,68 @@
 cmake_minimum_required(VERSION 2.8)
-project(TorusWorld) # Very tentative project name.
+project(dwfort)
```

Comment:
I understand that dwarf fortress is a source of inspiration, but we need a different name for our project before I can accept this commit.